### ISSUE TITLE
Update for WLP_LOGGING_CONSOLE_FORMAT "basic" in https://github.com/OpenLiberty/open-liberty-operator/blob/master/doc/user-guide.adoc

### ISSUE THREAD

**Issue by meiaus** [OpenLiberty/open-liberty-operator#158]
In the recent run with Open Liberty Operator, the OLA yaml setting with the following env resulted error as follows.
After updating to 'DEV' for 'basic', the error went away.

```
env:
    - name: WLP_LOGGING_CONSOLE_FORMAT
      value: basic
```
[WARNING ] CWWKG0032W: Unexpected value specified for property [consoleFormat], value = [basic]. Expected value(s) are: [DEV][SIMPLE][JSON]. Default value in use: basic. 
[ERROR ] CWWKG0075E: The value basic is not valid for attribute consoleFormat of configuration element logging. The validation message was: Value "basic" is not one of the options..

Please update the page: https://github.com/OpenLiberty/open-liberty-operator/blob/master/doc/user-guide.adoc in the sample snippet section as well as in the table (to provide the list of options, e.g. json, dev, simple.

![image](https://user-images.githubusercontent.com/27024803/82160381-8a1f3100-985a-11ea-8488-ee4aab47dac4.png)

**Comment by leochr** [comment]
Thanks @meiaus for bringing this to our attention.

@donbourne Do you know if there is a page we can point to from OL operator's user guide that has the list of allowed values for logging related environment variables (e.g. `WLP_LOGGING_CONSOLE_FORMAT`)? Thanks.

**Comment by donbourne** [comment]
we changed the value from basic to "dev" (same format, new name).  While it can produce that warning message in the event someone explicitly requests the basic format, the fallback/default when an unrecognized value is provided is basic/dev, so it has no affect on the behavior of the server.

we made the change to get rid of the confusion between the basic format for console.log and the basic format for messages.log, which aren't the same formats.  this was important because we added the ability for console.log to be formatted in the same way as the basic format of messages.log, and we realized this naming had to be fixed.

The expected values are documented at https://openliberty.io/docs/ref/config/logging.html
